docs/deployment/setup_gardener.md

```diff
@@ -1,44 +1,412 @@
+A Gardener landscape consists of several building blocks. The basis is an existing Kubernetes cluster which is referred to as *runtime* cluster.
+Within this cluster another Kubernetes cluster is hosted, which is referred to as *virtual Garden* cluster. It is nodeless, meaning it does not host any pods as workload. The virtual Garden cluster is the management plane for the Gardener landscape and contains resources like `Shoots`, `Seeds` and so on.
 
-In order to deploy the control plane components, please first deploy [`gardener-operator`](../concepts/operator.md#deployment) and create a [`Garden` resource](../concepts/operator.md#garden-resources).
+Any new Gardener landscape starts with the deployment of the Gardener Operator and the creation of a `garden` namespace on the designated runtime cluster.
 
-> [!CAUTION]
-> Below approach is deprecated and will be removed after v1.135 of Gardener has been released (around beginning of 2026).
+With the operator in place, the `Garden` resource can be deployed. It is reconciled by the Gardener Operator, which will create the "virtual" Garden cluster with its `etcd`, `kube-api-server`, `gardener-apiserver`, and so on.
+The API server is exposed through a LoadBalancer `Service` and a DNS record pointing to its external IP address is created.
 
-The [configuration values](../../charts/gardener/controlplane/values.yaml) depict the various options to configure the different components.
-Please consult [Gardener Configuration and Usage](../operations/configuration.md) for component specific configurations and [Authentication of Gardener Control Plane Components Against the Garden Cluster](./authentication_gardener_control_plane.md) for authentication related specifics.
+Once the `Garden` reports readiness, basic building blocks like `Cloudprofiles`, `ControllerDeployments`, and `ControllerRegistrations`, as well as `Secrets` granting access to DNS management for the internal domain and external default domain, are deployed to the virtual Garden cluster.
 
-Also, note that all resources and deployments need to be created in the `garden` namespace (not overridable).
-If you enable the Gardener admission controller as part of you setup, please make sure the `garden` namespace is labelled with `app: gardener`.
-Otherwise, the backing service account for the admission controller Pod might not be created successfully.
-No action is necessary if you deploy the `garden` namespace with the Gardener control plane Helm chart.
+To be able to host any `Shoots`, at least one `Seed` is required. The very first `Seed` is created via a `gardenlet` resource deployed to the virtual Garden cluster. The actual `gardenlet` `Pods` run on the runtime cluster or another Kubernetes cluster not managed by Gardener. Typically, this `Seed` is referred to as an __unmanaged__ `Seed` and is considered part of the Gardener infrastructure.
```

Comment:
Why is seed capitalized but gardenlet is not?
```suggestion
To be able to host any `Shoots`, at least one `Seed` is required. The very first `Seed` is created via a `Gardenlet` resource deployed to the virtual Garden cluster. The actual `gardenlet` `Pods` run on the runtime cluster or another Kubernetes cluster not managed by Gardener. Typically, this `Seed` is referred to as an __unmanaged__ `Seed` and is considered part of the Gardener infrastructure.
```

As there are other resources named `gardenlet`, e.g. the deployment, maybe the gardenlet resource should be mentioned with its full name `gardenlets.seedmanagement.gardener.cloud`.